crates/sui-indexer-alt-consistent-store/src/lib.rs

```diff
@@ -105,6 +109,20 @@ pub async fn start_service(
     )
     .await?;
 
+    // Register the rocksdb column family stats collector
+    registry
+        .register(Box::new(ColumnFamilyStatsCollector::new(
+            Some("rocksdb"),
+            indexer.store().db().clone(),
+            indexer.store().cf_names(),
+        )))
+        .context("Failed to register rocksdb column family stats collector")?;
+
+    tracing::info!(
+        "Periodic metrics reporting enabled for column families: {:?}",
+        indexer.store().cf_names()
+    );
```

Comment:
Let's move the registration inside `Store::open` -- that way we don't need to hold on to the `cf_names` any longer than we need to. This code only ever reads information you would get from that store.